index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <div class="videos">
+        <section class="video-section">
+            <video src="" controls width="1000"></video>
+            <article class="video-container">
+                 <div class="video-bottom-section">
+                     <a href="#">
+                        <img class="channel-icon" src=""/>
```

Comment:
- [ ] Also add an `img` to your `channel-icon` for example 😄 `<img class="channel-icon" src="http:/unsplash.it/250/150?gravity=center" />` 💯